src/scf-release/src/acceptance-tests-brain/test-resources/postgres-example-app/README.md

```diff
@@ -0,0 +1,3 @@
+# Postgres example app
+
+This is an example app initially intended to be used by the Postgres minibroker brain tests.
```

Comment:
Do we not have an existing sample app that can talk postgres?

Reply:
Now that I think about it… we have https://github.com/scf-samples/rails-example. Why the heck didn't I use that one for mariadb? I'm dumb. Sorry.